draft-ietf-quic-qpack.md

```diff
@@ -715,6 +714,132 @@ None.
+| 94    | access-control-allow-headers     | origin, x-requested-with, content-type, accept                                                         |
+| 95    | access-control-allow-headers     | dnt,x-customheader,keep-alive,user-agent,x-requested-with,if-modified-since,cache-control,content-type |
+| 96    | access-control-allow-headers     | dnt,x-mx-reqtoken,keep-alive,user-agent,x-requested-with,if-modified-since,cache-control,content-type  |
+| 97    | access-control-allow-methods     | options                                                                                                |
+| 98    | access-control-allow-methods     | get                                                                                                    |
+| 99    | access-control-allow-methods     | get, post, options                                                                                     |
+| 100   | access-control-allow-methods     | get, head, options                                                                                     |
+| 101   | access-control-allow-methods     | get, options                                                                                           |
+| 102   | origin                           |                                                                                                        |
+| 103   | x-frame-options                  | sameorigin                                                                                             |
+| 104   | x-frame-options                  | deny                                                                                                   |
+| 105   | timing-allow-origin              | \*                                                                                                     |
+| 106   | access-control-expose-headers    | content-length                                                                                         |
+| 107   | access-control-allow-credentials | TRUE                                                                                                   |
+| 108   | access-control-allow-credentials | FALSE                                                                                                  |
+| 109   | expect-ct                        |                                                                                                        |
+| 110   | content-security-policy          | script-src 'none'; object-src 'none'; base-uri 'none'                                                  |
+| 111   | upgrade-insecure-requests        | 1                                                                                                      |
+| 112   | x-served-by                      |                                                                                                        |
```

Comment:
While a very common header, it doesn't serve any purpose other than debugging. Maybe scrap it from here to encourage people to stop using it?

Reply:
+1